data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_xen_sshd.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm t="boolean">false</confirm>
+      <final_reboot t="boolean">true</final_reboot>
+      <second_stage t="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum t="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key t="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key t="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file t="boolean">true</accept_unsigned_file>
+      <accept_verification_failed t="boolean">true</accept_verification_failed>
+      <import_gpg_key t="boolean">true</import_gpg_key>
+    </signature-handling>
+  </general>
+  <report>
+    <errors>
+      <show t="boolean">true</show>
+      <timeout t="integer">20</timeout>
+      <log t="boolean">true</log>
+    </errors>
+    <messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">5</timeout>
+      <log t="boolean">true</log>
+    </messages>
+    <warnings>
+      <show t="boolean">true</show>
+      <timeout t="integer">10</timeout>
+      <log t="boolean">true</log>
+    </warnings>
+    <yesno_messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">30</timeout>
+      <log t="boolean">true</log>
+    </yesno_messages>
+  </report>
+  <bootloader>
+    <loader_type>grub2</loader_type>
+    <global>
+      <activate>true</activate>
+      <boot_mbr>true</boot_mbr>
+      <gfxmode>auto</gfxmode>
+      <os_prober>true</os_prober>
+      <terminal>console serial</terminal>
+      <serial>serial --unit=0 --speed=115200 --word=8 --parity=no</serial>
```

Comment:
Why do you change to use '--unit=0' at here?

Reply:
I checked most of the files for Xen, it was kept --unit=0 so I changed it to unit=0. I also checked the serial in the guide: [URL](https://doc.opensuse.org/documentation/leap/autoyast/single-html/book-autoyast/index.html#createprofile-sysconfig).
And at some places for KVM XML files, it was unit=0. For autoyast KVM profile and others, it was unit=1.
![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/46835794/b722052e-4e50-4bf4-839b-de9096fa32e2)

Reply:
> Why do you change to use '--unit=0' at here?

--unit=0 or --unit=1, anything major it will affect?
I hope it is ok.

Reply:
Noted, ty Julie, changed to unit=1
pushed commit